cmd/sops/subcommand/groups/add.go

```diff
@@ -0,0 +1,51 @@
+package groups
```

Comment:
nit: do we need a `subcommand` directory vs putting `groups` directly under `cmd/sops` ?